tests/k8s_testcases/example_policies/services1/services1-scheme.yaml

```diff
@@ -0,0 +1,102 @@
+namespaceList: ../../example_podlist/services1/ns_list.yaml
+podList: ../../example_podlist/services1/
+
+networkConfigList:
+  - name: ip
+    networkPolicyList:
+      - policies.yaml
+    expectedWarnings: 0
+
+  - name: np0
+    networkPolicyList:
+      - services1-networkpolicy0.yaml
+      - policies.yaml
+    expectedWarnings: 0
+
+  - name: np1
+    networkPolicyList:
+      - services1-networkpolicy1.yaml
+      - policies.yaml
+    expectedWarnings: 0
+
+  - name: np2
+    networkPolicyList:
+      - services1-networkpolicy2.yaml
+      - policies.yaml
+    expectedWarnings: 0
+
+queries:
+  - name: connectivity_map
+    connectivityMap:
+    - ip
+    - np0
+    - np1
+    - np2
+    expected: 0
+    outputConfiguration:
+      outputFormat: txt
+      fwRulesRunInTestMode: false
+    expectedOutput: ../expected_output/services1_connectivity_map.txt
+
+  - name: sanity
+    sanity:
+      - np1
+    expected: 2
```

Comment:
The output for this query is:
```
Running query sanity
NetworkConfig np1 failed sanity check:Ingress resource np1/default/demo-localhost/deny is redundant
Ingress rule no. 1 in Network policy np1/ingress-nginx/allow-all-within-namespace is redundant in np1
```
However it is somewhat awkward to define the Ingress resource as "redundant" - it's actually a "policy" that has no effect. Maybe "redundancy" should not be relevant for Ingress resources mixed with network policies? 
It might make sense to look for redundancy separately, once for Ingress and once for network policies.
In this case, I think there should be some warning for Ingress policy that has no effect due to network policy.